From c97c05f153b2d370fddf1e6da69350acecfa29e9 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <bochecha@daitauha.fr>
Date: Tue, 16 May 2017 22:11:35 +0200
Subject: [PATCH 1/4] Install the icons in the right folders

The paths were missing an "apps" subdir for it to be correct.

https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#install_icons
---
 share/icons/hicolor/32x32/{ => apps}/playitslowly.png    | Bin
 share/icons/hicolor/scalable/{ => apps}/playitslowly.svg |   0
 2 files changed, 0 insertions(+), 0 deletions(-)
 rename share/icons/hicolor/32x32/{ => apps}/playitslowly.png (100%)
 rename share/icons/hicolor/scalable/{ => apps}/playitslowly.svg (100%)

diff --git a/share/icons/hicolor/32x32/playitslowly.png b/share/icons/hicolor/32x32/apps/playitslowly.png
similarity index 100%
rename from share/icons/hicolor/32x32/playitslowly.png
rename to share/icons/hicolor/32x32/apps/playitslowly.png
diff --git a/share/icons/hicolor/scalable/playitslowly.svg b/share/icons/hicolor/scalable/apps/playitslowly.svg
similarity index 100%
rename from share/icons/hicolor/scalable/playitslowly.svg
rename to share/icons/hicolor/scalable/apps/playitslowly.svg
-- 
2.9.4

